MaxText/configs/base.yml

```diff
@@ -247,3 +247,28 @@ decode_sampling_temperature: 1.
 
 eval_interval: -1  # the specific number of train step between eval_step
 target_eval_loss: 0.  # early stop once reaching target eval_loss
+
+# Vertex AI Tensorboard Configurations
+# Set to True to create Vertex Tensorboard instance in GCP
+# Set to False if running via XPK as Vertex Tensorboard and Experiment will be created by XPK
+create_vertex_tensorboard: False
+# Project to create Vertex Tensorboard in if create_vertex_tensorboard=True
+# Project where Vertex Tensorboard to upload data exists if upload_data_to_tensorboard=True
+# Set to blank if running via XPK as XPK will set this as an environment variable
+vertex_tensorboard_project: ""
+# Location to create Vertex Tensorboard in if create_vertex_tensorboard=True
+# Location where Vertex Tensorboard exists if upload_data_to_tensorboard=True
+# Set to blank if running via XPK as XPK will set this as an environment variable
+# Vertex AI supported locations: https://cloud.google.com/vertex-ai/docs/general/locations#available-regions
+vertex_tensorboard_location: ""
+# Name of Vertex Tensorboard instance to create if create_vertex_tensorboard=True
+# Name of Vertex Tensorboard to use to upload data if upload_data_to_tensorboard=True
+# Set to blank if running via XPK as XPK will set this as an environment variable 
+vertex_tensorboard_name: ""
+# Name of Vertex Experiment to create if create_vertex_tensorboard=True
+# Name of Vertex Experiment to use to upload data if upload_data_to_tensorboard=True
+# Set to blank if running via XPK as XPK will set this as an environment variable
+vertex_experiment_name: ""
```

Comment:
I don't understand the Vertex Experiment terminology (and realistically neither will users). We need to explain it! (Like when do multiple runs go into the same experiment vs. different experiments.)

Reply:
This comment and some others might be handled in the README! But then let's link to the README and try to get one source of truth.

Reply:
Updated README with more details.